### ISSUE TITLE
Add CodeCov support.

### ISSUE THREAD

**Issue by scottrhoyt** [realm/SwiftLint#364]
Hey @jpsim. How do you feel about adding [CodeCov](https://codecov.io/) support to display code coverage metrics on the README and potentially provide another check on how PRs affect test coverage? I've done the travis integration before if you want me to take a look at it.

**Comment by jpsim** [comment]
That'd be great. What's your take on [coveralls](https://coveralls.io/) instead? I'm not familiar with CodeCov.

**Comment by scottrhoyt** [comment]
I evaluated them both for an in-house project pretty recently. I found them to be roughly feature comparable. I like that CodeCov seemed to have a few more details attached to PR comments, and that I got the sense it was more actively developed. I've also found it to be slightly better supported by my toolchain. However, I'm a bit biased now just because I'm more familiar with it. I'd be happy to go either direction. It might be nice to get different experience.
